formats/json/commonMain/src/kotlinx/serialization/json/JsonElementBuilders.kt

```diff
@@ -136,6 +137,14 @@ public class JsonArrayBuilder @PublishedApi internal constructor() {
         return true
     }
 
+    /**
+     * Adds the given JSON [elements] to a resulting JSON array.
+     *
+     * @return `true` if the list was changed as the result of the operation.
+     */
+    public fun addAll(elements: Collection<JsonElement>): Boolean =
```

Comment:
It's good this function's signature matches one from `MutableList` — we can add superclass later without breaking anything 👍🏻